test/appium/tests/test_wallet.py

```diff
@@ -71,4 +75,30 @@ def test_send_transaction_from_wallet(self, test, recipient, sender):
         chats.sign_transaction_button.click()
         chats.got_it_button.click()
         chats.verify_balance_is_updated(initial_balance_recipient, recipient_address)
+        if test == 'sign_now':
+            wallet = chats.wallet_button.click()
+            tr_view = wallet.transactions_button.click()
+            transaction = tr_view.transactions_table.find_transaction(amount=amount.replace(',', '.'))
+            details_view = transaction.click()
+            transaction_hash = details_view.get_transaction_hash()
+            verify_transaction_present(address=transaction_users[sender]['address'], transaction_hash=transaction_hash)
 
+    @pytest.mark.wallet
+    def test_balance_and_eth_rate(self):
+        errors = list()
+        home = HomeView(self.driver)
+        set_password_as_new_user(home)
```

Comment:
'set_password_as_new user' is redundant, since 'Recovery' is available from 'Console' view, now.

Reply:
Fixed.